index.html

```diff
@@ -0,0 +1,898 @@
+                    </header>
+                    <div class="section-three-header-two">
+                      <h4 class="padding-span title-span">$13m</h4>
+                      <h5 class="padding-span color-gray">Series b</h5>
+                    </div>
+                    <div class="image-div  ">
+                      <img src="./image/section-three-item1-img.jpg" alt="img-s-3">
+                    </div>
+                    <div class="section-three-header-sections ">
+                      <a href="#" ><i class="fas fa-bookmark s-3-h-s-a top"></i><span class="color-gray">
+                          Fiteness, +2
+                        </span></a>
+                        <br>
+                      <a href="#" ><i class="fas fa-briefcase s-3-h-s-a "></i>
+                        <span class="color-gray">
+                            Sequoia Capital
+                        </span>
+                      </a>
+                    </div>
+                    <div  class="section-three-header-sections ">
+                      <a href="#" class="s-3-h-s-a " > Asia Dealbook: Potato Play raises $1.75m from BEENEXT; PriceOye.pk</a>
+                      
+                      <a href="#" class="s-3-h-s-a " >raises pre-Series A <span class="color-gray"> on DealStreetAsia</span></a>
+                    </div>
+              </div>
+              <div class="section-three-item invicible">
+                  <header class="section-three-header">
+                      <span class="padding-span title-span">  Cecelia Health</span>
+                    </header>
+                    <div class="section-three-header-two">
+                      <h4 class="padding-span title-span">$13m</h4>
+                      <h5 class="padding-span color-gray">Series b</h5>
+                    </div>
+                    <div class="image-div  ">
+                      <img src="./image/section-three-item1-img.jpg" alt="img-s-3">
+                    </div>
+                    <div class="section-three-header-sections ">
+                      <a href="#" ><i class="fas fa-bookmark s-3-h-s-a top"></i><span class="color-gray">
+                          Fiteness, +2
+                        </span></a>
+                        <br>
+                      <a href="#" ><i class="fas fa-briefcase s-3-h-s-a "></i>
+                        <span class="color-gray">
+                            Sequoia Capital
+                        </span>
+                      </a>
+                    </div>
+                    <div  class="section-three-header-sections ">
+                      <a href="#" class="s-3-h-s-a " > Asia Dealbook: Potato Play raises $1.75m from BEENEXT; PriceOye.pk</a>
+                      
+                      <a href="#" class="s-3-h-s-a " >raises pre-Series A <span class="color-gray"> on DealStreetAsia</span></a>
+                    </div>
+              </div>
+            </div>
```

Comment:
- [ ] change the colors of the links

| Yours | Original |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/11758151/92157404-41e44880-ee22-11ea-8e18-668ff1e3876f.png) | ![image](https://user-images.githubusercontent.com/11758151/92157439-4f013780-ee22-11ea-9458-ce0ffd118add.png) |